test/udev-test.pl

```diff
@@ -1412,6 +1413,12 @@ sub udev_setup {
         chown (0, 0, $udev_dev) || die "unable to chown $udev_dev\n";
         chmod (0755, $udev_dev) || die "unable to chmod $udev_dev\n";
 
+        system("umount", "$udev_tmpfs");
+        system("rm", "-rf", "$udev_tmpfs");
```

Comment:
a simple "rmdir" would suffice here too, no? after all this should just be a mount point, hence a simple dir?

(I am just afraid of the effects "rm -rf" might have, and if we don't need it we should not use it i think..)